nixos/modules/services/networking/charybdis.nix

```diff
@@ -51,46 +51,55 @@ in
         '';
       };
 
+      motd = mkOption {
+        type = types.nullOr types.string;
```

Comment:
Not intentional, I just copied it from the extant option defs (not being aware of the deprecation; thanks!). "lines" sounds reasonable to me; I've changed it accordingly.